### ISSUE TITLE
Some disappointment about JPEG library (course 7)

### ISSUE THREAD

**Issue by deladan** [remisharrock/c-programming-with-linux-MOOC-issues-tracker#457]
Here's comment from a learner:  

> When I read :
> 
>     ... invoke the JPEG library (libjpeg.so), which can be used to read and write image files in JPEG format (feel free to play around here - this is pretty neat!)
> 
> I thought finally we leave the demo code ... BUT no ! How to 'play around here' ? No explanation, no example, nothing ...
> 
> And as said in this thread it is not easy to find by oneself ... It's a shame !

And I agree with him, activity is a little disappointing.  

Link to the activity:  https://courses.edx.org/courses/course-v1:Dartmouth_IMTx+DART.IMT.C.07+2T2018/courseware/f838e66a12864002be52ceaf73f2842f/7c4f5f4801354e60abefdab9d4f870eb/3?activate_block_id=block-v1%3ADartmouth_IMTx%2BDART.IMT.C.07%2B2T2018%2Btype%40discussion%2Bblock%40bc24378fda6340a097227d5b776942bd
Link to the thread: https://courses.edx.org/courses/course-v1:Dartmouth_IMTx+DART.IMT.C.07+2T2018/discussion/forum/c0a4c329d436b0bd49467e7723ad292feefab6f6/threads/5c3b6133db70a90987000d35

**Comment by chrissmiller** [comment]
I think this is probably more content related? @petraBT @remisharrock

**Comment by petraBT** [comment]
@remisharrock I tend to agree that the title of this assignment is misleading. We should come up with something more exciting for the future!
